.gitignore

```diff
@@ -39,7 +39,7 @@ bower_components
 build/Release
 
 # Dependency directories
-/node_modules/
+node_modules/
```

Comment:
we don't need the leading `/` anymore?  no more node_modules buried in the docs folder?